src/components/collapsible/Collapsible.js

```diff
@@ -68,12 +91,13 @@ class Collapsible extends React.Component<Props, State> {
                 } else if (!isOpen && onClose) {
                     onClose(this);
                 }
+                this.updateTransitionElementHeight();
```

Comment:
Does this need to happen after the previous setState call or can they be combined?